### PR TITLE
style: fix lint and style by biome

### PR DESCRIPTION
**Pull request by endes0**
This PR applies the changes made by biome formatter and linter. 
It is still a draft as there are +300 linter requested changes that has to be made manually.

### CODE REVIEW

**Review by oxcabe — CHANGES_REQUESTED** [review]
Configure Biome so it adds 2 spaces instead of 4. A fair amount of lines are way too long after the formatting, IMO.

**Comment by endes0** [comment]
> Configure Biome so it adds 2 spaces instead of 4. A fair amount of lines are way too long after the formatting, IMO.

We can also set a lineWidth

File: biome.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://biomejs.dev/schemas/1.6.3/schema.json",
+    "formatter": {
+      "indentStyle": "space"
+    }
+  }
```
Comment (oxcabe) [inline biome.json:6]: Add a newline at the end of files. You can configure VSCode to do this for you! 

See `files.insertFinalNewLine`: https://github.com/microsoft/vscode/blob/3648e8f381427e8b8352f8bf62e26b81cb3b7035/.vscode/settings.json#L109

File: public/locales/en/common.json
Comment (oxcabe) [inline public/locales/en/common.json:1]: This is out of the scope for this PR but I wont block it. Avoid doing this, though.

**Review by oxcabe — CHANGES_REQUESTED** [review]

File: biome.json (hunk 1 above)
Comment (endes0) [reply biome.json]: I thought biome fmt would automtically format their config file 😅. Fixed.

File: public/locales/en/common.json
Comment (endes0) [reply public/locales/en/common.json:1]: What are you referring? I had to add those new entries to the i18n as the linter was crying about the + - buttons that did not have aria labels.

**Comment by endes0** [comment]
BTW, @oxcabe before merging pls release version 2.3.1 (#91 ) as I think these are a lot of (internal) changes to be included to this version and also, while I tested this PR there is a possibility that something broke up, instead I'm sure that 2.3.1 is ready to go to production.

File: public/locales/en/common.json
Comment (oxcabe) [reply public/locales/en/common.json:1]: Oh, nevermind then, I thought you were adding i18n labels for an unrelated reason.

**Comment by oxcabe** [comment]
Closing this massive PR. We will instead opt for a gradual formatting/lint approach, managed from CI tooling such as reviewdog.

The branch should be preserved, though. I will lock the branch in order to avoid deleting it by mistake.

**Pull request closed without merging**